contracts/BaseJumpRateModelV2.sol

```diff
@@ -12,9 +13,9 @@ abstract contract BaseJumpRateModelV2 is InterestRateModel {
     uint256 private constant BASE = 1e18;
 
     /**
-     * @notice The address of the owner, i.e. the Timelock contract, which can update parameters directly
+     * @notice The address of the AccessControlManager contract
      */
-    address public owner;
+    IAccessControlManager public accessControlManager;
```

Comment:
Would it make sense to extend `AccessControlled`, to reuse the definitions we already have in that abstract class?

Reply:
It's not easy actually, because:
1) AccessControlled mixin is designed for upgradeable contracts, i.e. it has an initializer method and inherits Ownable2StepUpgradeable, rather than having a constructor and inheriting Ownable2Step.
2) JumpRateModels are created by factories, so the default owner would be the factory. We'll need some custom logic to transfer the ownership then.